README.md

```diff
@@ -165,11 +169,11 @@ The BigQuery to VCF pipeline is used to export variants in BigQuery to one VCF f
 Please refer to [BigQuery to VCF pipeline](docs/bigquery_to_vcf.md) for more
 details.
 
-## Running jobs in a particular region/zone
+## Running jobs in a particular region
```

Comment:
Now that this is always required the heading doesn't make a lot of sense I don't think

Reply:
Agree, I removed this section and added the following sentence to `GOOGLE_CLOUD_REGION` item.